packages/notifi-dapp-example/src/components/TopicStack.tsx

```diff
@@ -12,46 +12,65 @@ import {
   derivePrefixAndSuffixFromValueType,
   getFusionEventMetadata,
   isAlertFilter,
+  resolveAlertName,
+  useNotifiTenantConfigContext,
   useNotifiTopicContext,
 } from '@notifi-network/notifi-react';
 import React from 'react';
 
-type TopicStackProps = {
-  topicStackAlert: TopicStackAlert;
-  topic: FusionEventTopic;
+export type TopicStackProps = {
+  topicStackAlerts: TopicStackAlert[];
+};
+
+type InputParmValueMetadata = {
+  valueType: ValueType;
+  prefixAndSuffix: {
+    prefix: string;
+    suffix: string;
+  } | null;
 };
 
 export const TopicStack: React.FC<TopicStackProps> = (props) => {
   const { unsubscribeAlert, getAlertFilterOptions } = useNotifiTopicContext();
+  const { getFusionTopic } = useNotifiTenantConfigContext();
+  const benchmarkAlert = props.topicStackAlerts[0];
   const userInputOptions = React.useMemo(() => {
-    const input = getAlertFilterOptions(props.topicStackAlert.alertName)?.input;
+    const input = getAlertFilterOptions(benchmarkAlert.alertName)?.input;
     if (!input) return;
     return Object.values(input)[0];
   }, [getAlertFilterOptions]);
 
-  const fusionEventMetadata = getFusionEventMetadata(props.topic);
+  const { isLoading } = useNotifiTopicContext();
+  const resolved = resolveAlertName(benchmarkAlert.alertName);
+  const fusionTopic = getFusionTopic(resolved.fusionEventTypeId);
+  if (!fusionTopic) return null;
+  const fusionEventMetadata = getFusionEventMetadata(fusionTopic);
```

Comment:
As mentioned before, If we do not group the alerts with the same filterOptions, it will result in rendering multiple duplicate alerts. 
Not sure if I am missing anything?

Reply:
Yes we have the logic to filter out the duplicate alerts(with same filterOptions) in TopicStackRow